requirements.txt

```diff
@@ -3,6 +3,7 @@ cattrs==24.1.2
 certifi==2024.8.30
 charset-normalizer==3.4.0
 exceptiongroup==1.2.2
+GitPython
```

Comment:
We always pin versions in CHAINS project to stay protected of malicious releases. :)

Reply:
Fixed in #55 , thanks!